src/components/file/file.service.ts

```diff
@@ -103,25 +107,50 @@ export class FileService {
   ): Promise<File> {
     try {
       // TODO find a better way to check if object exists in s3 and move
+      let user: User | null;
+      const userSession: ISession = await this.authService.decodeAndVerifyToken(session.token);
+      user = (userSession.userId) ? await this.userService.readOne(userSession.userId, session) : null;
+
       const result = await this.db
         .query()
         .raw(
           `
-        MATCH (token:Token {active: true, value: $token})
+        MATCH
```

Comment:
There is a create abstraction in property-updater service see education service for how it is used.

Reply:
done